ts/client/features/transfer.ts

```diff
@@ -561,40 +584,25 @@ async function packBatch(
     };
     for (let i = 0; i < MAX_COMMIT_PER_BATCH; i++) {
         const pipe = pool.getNextPipe();
-        try {
-            const { commit, acceptedTxs, failedTxs } = await pack(
-                pipe,
-                storageManager,
-                params,
-                verifier
-            );
-            commitments.push(commit);
-            txBundle.acceptedTxs.push(...acceptedTxs);
-            txBundle.failedTxs.push(...failedTxs);
-        } catch (err) {
-            console.error(err);
-            continue;
```

Comment:
Rather than suppress errors, at this point I think we should let most things bubble to the top so we can discover and fix issues in the node and tests more easily. This would just give me an error message, now it gives a full stack and stops.